audio_builder.py

```diff
@@ -26,6 +27,12 @@
 cloudConfig = configparser.ConfigParser()
 cloudConfig.read('cloud_service_settings.ini')
 
+# Get the video file name and create the output folder based on the original video file name
+originalVideoFile = os.path.abspath(batchConfig['SETTINGS']['original_video_file_path'].strip("\""))
```

Comment:
I see some calls to the settings file with `original_video_file_path`, however, the config file remains unchanged, I think that creating a commit with these new settings and a default value would avoid some unexpected behavior on users that do not look into the code

Reply:
I think it's best to remove this for now